solutions/usaco-964.mdx

```diff
@@ -80,5 +80,53 @@ print(ans, file = open('whereami.out', 'w'))
 ```
 
 </PySection>
+	
+<CPPSection>
+	
+**Time Complexity:** $\mathcal{O}((N-K)^2)$
+
+```cpp
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <vector>
+#include <algorithm>
+#include <unordered_set>
+
+using namespace std;
+
+int main() {
+
```

Comment:
- No empty lines at the start and end of main
 - Tabs for indentation (as mentioned in the conventions)